### PR TITLE
Remove links inside the code block in the loaders.md.

### PR DESCRIPTION
**Pull request by 1010543618**
Remove links inside the code block.

I‘m confused about that does JS have a new syntax, when I frist see that. Finally I realized that's Markdown syntax.

### CODE REVIEW

**Comment by montogeek** [comment]
Those links are documentation, they are not render correctly at this moment but that's another issue.

**Comment by 1010543618** [comment]
@montogeek 

Oops, I made some mistake, you already implemented this feature. [remark-refractor/js-links-details.js at ed597eed49f792354e8b9889f001a3f2f5aa24d3 · montogeek/remark-refractor](https://github.com/montogeek/remark-refractor/blob/ed597eed49f792354e8b9889f001a3f2f5aa24d3/src/js-links-details.js)

Should we change `\src\content\concepts\loaders.md`, `src\content\configuration\module.md` to mdx?

**Comment by 1010543618** [comment]
I use global search to search `js-with` find those two file has *md* suffix, others are *mdx* suffix.

**Comment by montogeek** [comment]
@1010543618 I think that would work, Can you try please?

File: src/content/concepts/loaders.md [hunk 1]
```diff
@@ -65,14 +65,14 @@ module.exports = {
       {
         test: /\.css$/,
         use: [
-          { loader: ['style-loader'](/loaders/style-loader) },
+          { loader: 'style-loader' },
```
Comment (1010543618) [inline src/content/concepts/loaders.md]:
```
{ loader: <link "/loaders/style-loader">
	<default>
		'style-loader'
	</default>
</link> },

<link "/loaders/style-loader">
	<default>
		{ loader: 'style-loader' },
	</default>
</link>
```

Comment (1010543618) [reply src/content/concepts/loaders.md]: ang change `js-with-links-with-details` to `js-with-links-details`

**Comment by 1010543618** [comment]
@montogeek 
According to my test, Links in `js-with-links` can be transformed  to HTML a tag only when it in the comment. (I think this it right way, not a bug.)
So links in `src\content\configuration\module.md` can be fixed by using mdx loader, but for `\src\content\concepts\loaders.md` we can fix it like this [Remove links inside the code block in the loaders.md. by 1010543618 · Pull Request #2857 · webpack/webpack.js.org](https://github.com/webpack/webpack.js.org/pull/2857/files#r264669389), because links in it did not in the comment.

**Comment by montogeek** [comment]
Please apply your suggested changes

**Pull request closed without merging**